Project3_Abasore.py

```diff
@@ -0,0 +1,112 @@
+
+import requests
+import string
+
+
+"""
+Project Plans: to sort the requencies of words for each book by Jane Austen. Then, adjust with ratio for size of book. 
+Compare her most frequent words from the beginng,middle, and end of her career. 
+"""
+
+
+def adjust_factor(max, freq):
+	"""
+	take frequency and max frequency as an input and makes an adjustment ratio and stores it in a new dictionary
+	"""
+	output = {}
+	for w , v in freq.items():
+		output[w] = v/max[0]
+	return output
+
+def skip_first(text):
+	for line in text:
+		line = line.replace('-',' ')
+		line = line.replace('_',' ')
+		if line.startswith('*** START'):
+			break
+
+def clean_up_and_sort(filename, skip):
+	"""
+	removes the header from a text file and cleans out all the /n and like characters.
+
+	"""
+	#using the string method of startwith to find the end of the header
+	fin = open(filename)
+	hist = {}
+	if skip:
+		skip_first(fin)
+	
+	for line in fin:
+		#this is the basic function that makes the histogram
+		line = line.replace('-',' ')
+		line = line.replace('_',' ')
+		to_take_off = string.punctuation + string.whitespace
+		for word in line.split():
+ 			#this divides each word up at the spaces, you can put anything you want to split at, a helpful one would be the \n new line
+			word = word.strip(to_take_off)#strip only takes it off the end
+			word = word.lower() #makes it lower case
+			hist[word] = hist.get(word, 0) + 1 #this is a simple way to count the frequencies. get has default to what you se it equal to (this case 0) unless it already has a value. no matter what though, it gets a +
+	return hist
+		
+
+def count_freq(hist, number = 10 ):
+	"""
+	countes the frequency of each word and stores it in a dictionary with the word as the key and the frequency as the
+	value. A second dicionary is make the stores the word as the key and teh ratio as the value.
+	and ratio as the second value.
+	"""
+	output = []
+	for w, v in hist.items():
+		output.append((v, w)) #Here I am just making a list of all the pairs in the dictioanry, this will keep it so every word is only done once
+		output.sort() #this updates the current value of output to be in order
+		output.reverse() #this makes it so you get the highest number first
+	new = adjust_factor(output[0], hist)
```

Comment:
from a readability standpoint, it would make more sense to pass in the max frequency instead of the tuple containing the max frequency. Again, doesn't really impact performance.